packages/teleport/src/components/DownloadLinks/DownloadLinks.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2022 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Box, Link } from 'design';
+import getDownloadLink from 'teleport/services/links';
+import useTeleport from 'teleport/useTeleport';
+
+export default function DownloadLinks({ version }: Props) {
+  const ctx = useTeleport();
```

Comment:
We only care about `isEnterprise` from the context, so it is better to just add it as a prop to this component rather than retrieving the entire context. This will also prevent unit tests from breaking due to the unretrievable context. 

eg. for `AddDatabase` you would just get the `isEnterprise` flag from the ctx in `useDatabases` and then pass it down via `Databases` -> `AddDatabase` -> `DownloadLinks`